.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Build on various envs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize # the head branch is updated from the base branch, new commits are pushed to the head branch, or the base branch is changed
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - 'release/**'
+
+jobs:
+  build:
+    name: build
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [ '14', '16' ]
```

Comment:
Can you please add an issue that it does not build with Node 17 and 18? We should look into that in the next release.

Reply:
#996